lib/OpenQA/WebAPI/Controller/API/V1/Job.pm

```diff
@@ -95,9 +93,9 @@ sub list {
     $validation->optional('scope')->in('current', 'relevant');
     $validation->optional('limit')->num(0);
     $validation->optional('latest')->num(1);
-
-    my $limit = $validation->param('limit') // JOB_QUERY_LIMIT;
-    return $self->render(json => {error => 'Limit exceeds maximum'}, status => 400) unless $limit <= JOB_QUERY_LIMIT;
+    my $limit = $validation->param('limit');
```

Comment:
why is it not necessary to set a default here anymore?

Reply:
Do we want each route to have a `limit` argument, a default limit and a maximum? Currently they're not consistent.